### PR TITLE
[e2e] Add check for crc-admin context set

### PR DESCRIPTION
**Pull request by praveenkumar**
This patch will make sure the context feature is working as expected.

fixes: #1966

### CODE REVIEW

File: test/e2e/features/basic.feature [hunk 1]
```diff
@@ -125,7 +125,7 @@ Feature: Basic test
     Scenario: Monitoring stack check
         Given checking that CRC is running
         When executing "eval $(crc oc-env)" succeeds
-        And login to the oc cluster succeeds
+        And Set kubeconfig context to "crc-admin" succeeds
```
Comment (cfergeau) [inline test/e2e/features/basic.feature:128]: `And setting kubeconfig context to "crc-admin" succeeds` reads better

**Review by cfergeau — COMMENTED** [review]
It seems to be macOS specific, but might be worth testing for https://github.com/code-ready/crc/issues/1569 too?

File: test/e2e/features/basic.feature [hunk 2]
```diff
@@ -135,7 +135,7 @@ Feature: Basic test
     Scenario: Monitoring stack check
         Given checking that CRC is running
         And executing "crc oc-env | Invoke-Expression" succeeds
-        And login to the oc cluster succeeds
+        And Set kubeconfig context to "crc-admin" succeeds
```
Comment (guillaumerose) [inline test/e2e/features/basic.feature:138]: I am wondering how the CI is passing if this step is not defined anymore 🤔

Comment (praveenkumar) [reply test/e2e/features/basic.feature:138]: @guillaumerose because it was for windows platform so doesn't run on any CI.

**Pull request merged**